esp/mlrs-wifi-bridge/mlrs-wifi-bridge-boards.h

```diff
@@ -192,7 +236,7 @@ GPIO15 = RTC_GPIO13
 //-- M5Stack ATOM Lite
 #elif defined MODULE_M5STACK_ATOM_LITE
     #ifndef ARDUINO_M5Stack_ATOM // ARDUINO_BOARD != ARDUINO_M5Stack_ATOM
-	      #error Select board ARDUINO_M5Stack_ATOM!
+	      #error Select board M5Stack-ATOM!
```

Comment:
this appears to be the correction of a bug unrelated to this PR. Maybe separate out. It seems to me that there are more of this kind of bugs, e.g. in line 141. We probably want to go over all the #error and get them right in one stroke, and a separate PR.

Reply:
Done.